runtimes/php/plugin.yaml

```diff
@@ -0,0 +1,29 @@
+version: 0.1
+downloads: # Needed for the php runtime to install packages
+  - name: composer
+    executable: true
+    version: 2.5.1
+    downloads:
+      - version: 2.5.1
+        gpg: CBB3D576F2A0946F
```

Comment:
Is this missing a URL or is this sufficient?

Reply:
this is sufficient - we are just using the gpg key to allow another php runtime tool to download this